### DISCUSSION TITLE
how to enable wireless debugging in flutter

### DISCUSSION THREAD

**Discussion by shettysnehal** [community/community#133516] [Mobile] [answered]
### Select Topic Area

Question

### Body

fliutter

**Comment by Floyden-Monteiro** [comment]
To enable wireless debugging for Flutter, you need to set up ADB (Android Debug Bridge) over Wi-Fi. Here’s a step-by-step guide to help you:

Connect your device via USB:

Connect your Android device to your computer using a USB cable.
Ensure that USB debugging is enabled on your device. You can enable it from Settings > Developer options > USB debugging.

**Comment by Floyden-Monteiro** [comment, marked as answer]
To enable wireless debugging for Flutter, you need to set up ADB (Android Debug Bridge) over Wi-Fi. Here’s a step-by-step guide to help you:
#133517 
Connect your device via USB:

Connect your Android device to your computer using a USB cable.
Ensure that USB debugging is enabled on your device. You can enable it from Settings > Developer options > USB debugging.

**Comment by erenmalkoc** [comment]
On your Android device, navigate to Settings > Developer options.
Enable Wireless debugging if it's available. This option is typically found in Android 11 and above.
